src/screens/DataApi/DataApi.style.tsx

```diff
@@ -25,6 +25,11 @@ export const GreenLinkButton = styled(LinkButton)`
   }
 `;
 
+export const BlueLinkButton = styled(LinkButton)`
+  color: ${COLOR_MAP.BLUE};
+  text-transform: none;
+`;
```

Comment:
Consider using the `ExternalLink` component instead. This is mostly okay as the LinkButton is actually a link in the DOM, but the hover state makes it look like a button.

Ideally, we want to use buttons for actions and links for navigations, but we don't always follow that.